core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -646,126 +617,6 @@ public void testMultiValuedFieldOrderedByKeyDesc() throws Exception {
     }
 
     /**
-     * The script will change to document date values to the following:
-     * <p>
-     * doc 1: [ Feb 2, Mar 3]
-     * doc 2: [ Mar 2, Apr 3]
-     * doc 3: [ Mar 15, Apr 16]
-     * doc 4: [ Apr 2, May 3]
-     * doc 5: [ Apr 15, May 16]
-     * doc 6: [ Apr 23, May 24]
-     */
-    public void testMultiValuedFieldWithValueScript() throws Exception {
-        SearchResponse response = client().prepareSearch("idx")
-                .addAggregation(dateHistogram("histo")
-                        .field("dates")
-                        .script(new Script("new DateTime(_value, DateTimeZone.UTC).plusMonths(1).getMillis()"))
```

Comment:
I re-added those tests and another mock. Decided to also keep the date math so the existing tests don't have to be adapted. Not sure though if the way I mocked the script for values here is making this test the right thing and I would be glad if you had another look at it.